docs/runbooks/NoReadyVirtController.md

```diff
@@ -2,51 +2,52 @@
 
 ## Meaning
 
-From a high-level perspective the virt-controller is responsible for all the cluster wide virtualization functionality.
+This alert fires when no available `virt-controller` devices have been detected for 5 minutes.
```

Comment:
Please change "5" to "five". ISG says to spell out numbers that are less than 10.

Reply:
True, but it also says that measurements should always use numerals, which (AFAICT) is the case here.

Reply:
Minutes are considered measurements?! Interesting. I'll look into it.